<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>666</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>666</width>
     <height>800</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="GameArea" name="widgetGameArea" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>350</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetBGD" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>666</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>666</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>666</width>
      <height>800</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/pic/bg.jpg);</string>
    </property>
   </widget>
   <widget class="NextArea" name="widgetNextArea" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>170</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelN">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetN" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>190</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetG" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>370</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetG_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>350</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetN_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>170</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetS" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>190</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelS">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetS_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetH" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>190</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetH_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>170</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelN_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>HOLD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="HoldArea" name="widgetHoldArea" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>170</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetSp" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>190</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(247,247,247,0.5);border-radius:15px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSp">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>SPEED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(247, 247, 247);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetSp_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>170</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>720</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(247,247,247,0.5);border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <zorder>widgetBGD</zorder>
   <zorder>labelN</zorder>
   <zorder>widgetN</zorder>
   <zorder>widgetG</zorder>
   <zorder>widgetG_2</zorder>
   <zorder>widgetGameArea</zorder>
   <zorder>widgetN_2</zorder>
   <zorder>widgetNextArea</zorder>
   <zorder>widgetS</zorder>
   <zorder>labelS</zorder>
   <zorder>widgetS_2</zorder>
   <zorder>labelScore</zorder>
   <zorder>widgetH</zorder>
   <zorder>widgetH_2</zorder>
   <zorder>labelN_2</zorder>
   <zorder>widgetHoldArea</zorder>
   <zorder>widgetSp</zorder>
   <zorder>labelSp</zorder>
   <zorder>widgetSp_2</zorder>
   <zorder>labelSpeed</zorder>
   <zorder>Pause</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NextArea</class>
   <extends>QWidget</extends>
   <header>nextarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameArea</class>
   <extends>QWidget</extends>
   <header>gamearea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HoldArea</class>
   <extends>QWidget</extends>
   <header>holdarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
